### PR TITLE
Added iot-device module + added the dependency for pi4j

### PR DESCRIPTION
**Pull request by danielvdwal**
@matthiasbalke @pDiller I have added the client module for the raspberrypi, I figured it should be called iot-device.
I also added a dependecy for pi4j and included the module in the settings.gradle

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -2,4 +2,5 @@ rootProject.name = 'JiraAlerts'
 
 include 'integration'
 include 'jira-component'
-include 'iot-controller'
\ No newline at end of file
+include 'iot-controller'
+include 'iot-device'
```
Comment (pDiller) [inline settings.gradle]: why exactly is iot device added in here? what do you want to do with it?

Comment (danielvdwal) [inline settings.gradle]: iot-device is not going to be built, if it is not mentioned in the settings.gradle

Comment (pDiller) [inline settings.gradle]: yes, you´re right. But what is the task of the module? Whats the difference between iot-controller and iot-device?

Comment (danielvdwal) [inline settings.gradle]: iot-controller is supposed to be on the server side, iot-device is the module that should be deployed on the pi itself.

Comment (pDiller) [inline settings.gradle]: ah okay, so iot-device is holding the rest-api for the raspberry?

Comment (danielvdwal) [inline settings.gradle]: yes, as well as the "native" logic for controlling the gpio pins via pi4j

Comment (pDiller) [inline settings.gradle]: Okay, i will approve the pullrequest, ;-)

**Pull request merged**